client/extensions/woocommerce/app/store-stats/constants.js

```diff
@@ -1,3 +1,41 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+export const topProducts = {
+	title: translate( 'Products' ),
+	values: [
+		{ key: 'name', title: translate( 'Title' ) },
+		{ key: 'price', title: translate( 'Price' ) },
+		{ key: 'total', title: translate( 'Sales' ) },
+	],
+	empty: translate( 'No products found' ),
+	statType: 'statsTopEarners',
+};
+
+export const topCategories = {
+	title: translate( 'Categories' ),
+	values: [
+		{ key: 'name', title: translate( 'Title' ) },
+		{ key: 'quantity', title: translate( 'Quantity' ) },
+		{ key: 'total', title: translate( 'Total' ) },
+	],
+	empty: translate( 'No categories found' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **34** times:
`translate( 'No categories found.' )` *ES Score: 14*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/wcss-listview-routes)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).